.circleci/config.yml

```diff
@@ -29,7 +29,8 @@ jobs:
         type: string
     working_directory: /opt/app-root/zync
     docker:
-      - image: registry.access.redhat.com/ubi7/ruby-27
+      # A Zync image tagged manually for CI tests
+      - image: quay.io/3scale/zync:ci-builder
```

Comment:
Is this base image still temporary?

Reply:
No, me and @akostadinov discussed this and decided that this is a reasonable approach.

Basically, the problem is that `ubi9/ruby-31` misses some dependencies required to run zync: irb, rdoc on Ruby side, but also PostgreSQL client.

So, we considered various solutions:

1. Create a separate repository for building the image for Zync CI, like porta has: https://github.com/3scale/system-builder
We considered this is an overkill

2. (this approach) Just use a manually tagged Zync image for CI

3. Create a separate Dockerfile in zync repo for building an image for CI, and push it to quay. Potentially, CI itself could build this image

4. Make the docker build step (from the existing Dockerfile) a dependency for the rest of the tests.

2 seemed the most simple and effective.